burgle.lic

```diff
@@ -422,25 +421,29 @@ class Burgle
     #if you've seen footsteps, don't bother storing items, and just get out if in safe mode
     return if Flags['burgle-footsteps']
     #store loot in your loot container, unless the item in your hands is a rope or lockpick
-    if DRC.right_hand
-      if GameObj.right_hand.noun =~ /lockpick|rope/
-        stow_hand("right")
-      else
-        temp_loot = DRC.right_hand
-        @loot_list.push(temp_loot) if put_item?(DRC.right_hand)
-      end
+    case DRC.right_hand
+    when nil
+    when /(?:lockpick|rope)$/
+      stow_hand("right")
+    when *@burgle_settings['item_whitelist']
+      put_item?(DRC.right_hand)
+    else
+      temp_loot = DRC.right_hand
+      @loot_list.push(temp_loot) if put_item?(DRC.right_hand)
```

Comment:
Courtesy of VTCifer, only add items to the "loot list" if they aren't in the `item_whitelist`. This protects those items from being dropped/pawned/binned.